### DISCUSSION TITLE
LLM model

### DISCUSSION THREAD

**Discussion by paramjitsingh006** [AltimateAI/vscode-dbt-power-user#1466] [General]
While using power user, is it possible to specify our own LLM model? e.g. in our organization, only snowflake Cortex and Copilot are the approved tools.
So instead of using default LLM model used by power user, can we override to use snowflake cortex?

**Comment by ntantri** [comment]
+1 and some more context.

Similar, but different org, we have limitations because of data privacy and would like to understand if there is any investment in allowing other LLM models/APIs.
